### PR TITLE
EPMDJ-9148 Update to Kotlin 1.5  add memory test

### PR DESCRIPTION
**Pull request by Mazel-Tovr**

### CODE REVIEW

File: src/main/kotlin/Core.kt [hunk 1]
```diff
@@ -175,14 +179,25 @@ suspend inline fun <reified T : Any> Transaction.store(any: T, schema: String) {
         val simpleName = T::class.toTableName()
         prepareTable(schema, simpleName)
         dbContext.set(schema)
-        val json = json.encodeToString(T::class.serializer(), any)
-        val stmt =
-            """
-            |INSERT INTO $schema.${simpleName.toLowerCase()} (ID, JSON_BODY) VALUES ('${idValue.hashCode()}', '$json')
+        val measureTime = measureTime {
+            val file = File.createTempFile("prefix-", "-suffix")
```
Comment (Mazel-Tovr) [inline src/main/kotlin/Core.kt:183]: add todo

File: src/test/kotlin/MemoryTest.kt [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import org.jetbrains.exposed.sql.statements.jdbc.*
+import org.jetbrains.exposed.sql.transactions.*
+import org.junit.jupiter.api.*
+import org.junit.jupiter.api.Test
+import java.io.*
+import java.util.*
+import kotlin.test.*
+
+@Disabled
+class MemoryTest : PostgresBased(schema) {
```
Comment (Mazel-Tovr) [inline src/test/kotlin/MemoryTest.kt:32]: add test

File: src/main/kotlin/Core.kt (hunk 1 above)
Comment (Mazel-Tovr) [reply src/main/kotlin/Core.kt:183]: add annotation

File: src/test/kotlin/MemoryTest.kt [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import org.junit.jupiter.api.*
+import org.junit.jupiter.api.Test
+import kotlin.test.*
+
+@Disabled
+class MemoryTest : PostgresBased(schema) {
+
+    // OOM With such config with `string memory test` and all ok with `stream memory test`
+    // Tested with heap size = 1g
+    private val size = 200000
```
Comment (LikhanovMaxim) [inline src/test/kotlin/MemoryTest.kt:28]: 200_000

File: src/test/kotlin/MemoryTest.kt [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+import kotlinx.coroutines.*
+import org.junit.jupiter.api.*
+import org.junit.jupiter.api.Test
+import kotlin.test.*
+
+@Disabled
+class MemoryTest : PostgresBased(schema) {
+
+    // OOM With such config with `string memory test` and all ok with `stream memory test`
+    // Tested with heap size = 1g
```
Comment (LikhanovMaxim) [inline src/test/kotlin/MemoryTest.kt:27]: I think this comment should be with @Disabled.
Or disable only this test "save object as string memory test`" and move this comment there

Comment (Mazel-Tovr) [reply src/test/kotlin/MemoryTest.kt:27]: I prefer first option, because for these tests we need to set heap with 1g size

**Pull request merged**